CHANGELOG.asciidoc

```diff
@@ -45,6 +45,17 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha5...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Periodically log internal metrics. {pull}1955[1955]
```

Comment:
Yes, it looks like "git rebase master" didn't do a good job. They were part of alpha5. Thanks for noticing it. I am fixing it manually.